docs/users/cli-usage.md

```diff
@@ -668,7 +668,8 @@ ddev restore-snapshot d8git_20180801132403
 Restored database snapshot: /Users/rfay/workspace/d8git/.ddev/db_snapshots/d8git_20180801132403
 ```
```

Comment:
In the above restore examples, need to  mention and show what will rapidly become the default technique, 

<img width="490" alt="rfay_ddevprod___" src="https://user-images.githubusercontent.com/112444/103559742-dbc10e00-4e73-11eb-932f-b70b3e9daa06.png">